programs/sbf/rust/sibling_inner_instructions/Cargo.toml

```diff
@@ -9,7 +9,13 @@ license = { workspace = true }
 edition = { workspace = true }
 
 [dependencies]
+solana-account-info = { workspace = true }
+solana-instruction = { workspace = true, features = ["syscalls"] }
+solana-msg = { workspace = true }
 solana-program = { workspace = true }
```

Comment:
You should be able to remove this now since the helpers from solana-instruction are being used